### PR TITLE
Update pom file.

### PR DESCRIPTION
**Pull request by ypriverol**
New version of jmzIdentML. 1.2.10 PR.

### CODE REVIEW

**Comment by ypriverol** [comment]
Release has been done: https://github.com/PRIDE-Utilities/jmzIdentML/issues/7

**Comment by julianu** [comment]
The release of jmzidentml does unfortunately not really help.
The original version in the PR https://github.com/PRIDE-Utilities/jmzIdentML/pull/3 did massively improve jmzidentml performance, but the current release 1.2.10 does unfortunately not. Running the test MzIdentMLExportAndImportTest takes 18s with release 1.2.5 and 240s with >=1.2.6 (including 1.2.10).

I did not have time to check why, but will try to have a look later. For now, I have build issues with this part of the jmzidentml pom, when trying to build the GitHub code directly:

    <parent>
        <groupId>uk.ac.ebi.pride.architectural</groupId>
        <artifactId>pride-core</artifactId>
        <version>1.0.1</version>
    </parent>

Basically, I cannot find the dependency. Is it available somewhere?

**Comment by ypriverol** [comment]
Here the pride-core project. You can build it locally.

**Comment by ypriverol** [comment]
New changes have been implemented to the way we generate the `hashCode` on every important object without needs to generate an object inside the `hashCode`. See this discussion about the performance of https://stackoverflow.com/questions/5038204/apache-commons-equals-hashcode-builder

**Comment by ypriverol** [comment]
@julianu This has been fixed locally in my machine and I will commit a new version soon. We need to release first all the dependencies.
